web/src/containers/Trade/_TradeContainer.scss

```diff
@@ -56,6 +57,22 @@
 	font-size: 1.1rem;
 }
 
+.vertical-line-seperator {
+	width: 1px;
+	height: 50%;
+	margin: 0.4rem 0.33rem;
+	border: 1px solid rgba(255, 255, 255, 0.4);
```

Comment:
It is better not to hard-code colors. We already have colors defined for different border shades.